stage3-py.sh

```diff
@@ -16,80 +16,13 @@ fi
 # Never allow the installation to upgrade rubin_env.  Generally enforcing
 # the pin is only needed for releases, where the current version may have
 # moved ahead.
+#
 rubin_env_ver=$(mamba list rubin-env$ --no-banner --json \
 		    | jq -r '.[0].version')
 # Do the rest of the installation
 mamba install --no-banner -y \
       "rubin-env==${rubin_env_ver}" \
```

Comment:
You can remove this line since [-rsp includes the base](https://github.com/conda-forge/rubinenv-feedstock/blob/main/recipe/meta.yaml#L216).